### PR TITLE
GCI-38 Edit Account Page

### PR DESCRIPTION
**Pull request by chaityabshah**
[Melange task](http://www.google-melange.com/gci/task/view/google/gci2014/4955787038294016)
[Ticket](https://issues.openmrs.org/browse/GCI-38)
Hey @elliottwilliams I was wondering if you wanted anything else for the widgets.... You said a plain panel was enough, but I wasn't sure what you wanted inside. Let me know if I can improve that template widget so it can be used in other page mockups or something.

### CODE REVIEW

File: app/system-modules/profile/views/edit-profile.ejs [hunk 1]
```diff
@@ -1,121 +1,14 @@
...
-                </a>
-            <% } %>
-
-        <% } %>
-
-    </li>
-    <% }) %>
-
-</ul>
+<div class="col-md-5">
```
Comment (elliottwilliams) [inline app/system-modules/profile/views/edit-profile.ejs]: I think you need to use `col-md-6` here and below. Right now, the grid items look like they don't expand to the full width of their container.

**Comment by elliottwilliams** [comment]
Take a look at my comment, and the notification like we discussed in IRC. Otherwise, great!

**Comment by chaityabshah** [comment]
Applied the changes, let me know how you think panel-info looks, Maybe panel-warning or panel-danger might be better for this purpose?

**Comment by elliottwilliams** [comment]
Nice. What if you used `panel-primary` with a header (http://getbootstrap.com/components/#panels-heading). Then instead of having the "Welcome to the OpenMRS Community!" title as a `<h2>`, you would use it as the panel heading. You'd also want to make the buttons `primary` color.

The reason I like this is that the `primary` style is really high-contrast with the off-white background of OpenMRS ID, so it helps the notification callout standout from other content.

Sound good to you?

**Comment by chaityabshah** [comment]
Looks nice, thanks for the suggestions :)

**Comment by elliottwilliams** [comment]
Nice. Thanks!

> This task has been approved, but will not be merged into the repository until all GCI tickets in the ID Dashboard UI Revamp project have been submitted. We're doing this so that multiple students can submit their design implementation and we can select from the best collective implementation.

**Pull request closed without merging**